.docs/content/1.guide/5.cpp.md

```diff
@@ -0,0 +1,126 @@
+# Compilation steps for cpp API
+
+## Compilation of ArmoniK.Api.cpp Client and Server on Linux using Docker
+
+In order to compile the client and server on Linux, we use a Docker image to set up the necessary environment and dependencies. This ensures a consistent and clean environment for compilation.
+
+### Prerequisites Linux
+
+1. Install Docker on your Linux system. Follow the instructions on the [official Docker documentation](https://docs.docker.com/engine/install/).
+
+2. Clone the repository containing the source code and the necessary scripts.
+
+### Compilation Steps for Linux
+
+1. Open a terminal in the root directory of the cloned repository.
+
+2. Run the `compile.sh` script:

```

Comment:
Could you add more info about the location of this file. If I strictly follow your guide, I expect that `./compile` is a file at the root of the project. That's not the case.